src/clj/athenaeum/handlers/book.clj

```diff
@@ -1,9 +1,15 @@
 (ns athenaeum.handlers.book
   (:require [ring.util.response :as response]
-            [athenaeum.domain.book :as books]
-            [athenaeum.db :as db]))
+            [athenaeum.session.core :as session]
+            [athenaeum.db :as db]
+            [athenaeum.domain.book :as books]))
 
 (defn fetch
-  [_]
-  (db/with-transaction [tx @db/datasource]
-    (response/response (books/fetch-all tx))))
+  [{:keys [cookies]}]
+  (if-let [session-id (get cookies "session-id")]
```

Comment:
This checking should go into a middleware. Then it can be maintained and tested separately.